kayenta-web/src/main/java/com/netflix/kayenta/controllers/CanaryConfigController.java

```diff
@@ -190,11 +185,11 @@ private static void validateNameAndApplicationAttributes(@RequestBody CanaryConf
   }
 
   @ApiOperation(value = "Delete a canary config")
-  @RequestMapping(value = "/{canaryConfigId:.+}", method = RequestMethod.DELETE)
+  @ResponseStatus(HttpStatus.NO_CONTENT)
```

Comment:
Is this a breaking change?

Reply:
no, this was already there before. I extracted this to the annotation only